src/arch/ofi/machine.C

```diff
@@ -87,37 +160,84 @@
 #include <rdma/fi_tagged.h>
 #include <rdma/fi_rma.h>
 
-#define USE_OFIREQUEST_CACHE 0
+#define USE_OFIREQUEST_CACHE 1
 
 /* Definition of OFIRequest + request cache */
 #include "request.h"
 
 /* Runtime to exchange EP addresses during LrtsInit() */
-#if CMK_USE_PMI || CMK_USE_SIMPLEPMI
+/* someday, we'll update to pmix, today is not that day */
+#define CMK_USE_CRAYPMI2 1
+#if CMK_USE_CRAYPMI2
+#include "runtime-craypmi2.C"
+#elif CMK_USE_CRAYPMI
+#include "runtime-craypmi.C"
+#elif CMK_USE_PMI || CMK_USE_SIMPLEPMI
 #include "runtime-pmi.C"
 #elif CMK_USE_PMI2
 #include "runtime-pmi2.C"
 #elif CMK_USE_PMIX
 #include "runtime-pmix.C"
 #endif
 
+/* this should be set for us during the build, but cmake confounds again */
+#define CMK_CXI 1
```

Comment:
Should probably be fixed

Reply:
Latest push has changes to build and buildold to address this.